### PR TITLE
fix: DSTART on replay 

### PR DESCRIPTION
**Pull request by Mryashbhardwaj**
derive `schedule time` by `execution_date` in airflow 2.1.4

### CODE REVIEW

File: ext/scheduler/airflow/dag/expected_dag.2.1.py [hunk 1]
```diff
@@ -102,7 +102,6 @@ def get_entrypoint_cmd(plugin_entrypoint_script):
     k8s.V1EnvVar(name="JOB_NAME", value='infra.billing.weekly-status-reports'),
     k8s.V1EnvVar(name="OPTIMUS_HOST", value='http://optimus.example.com'),
     k8s.V1EnvVar(name="PROJECT", value='example-proj'),
-    k8s.V1EnvVar(name="SCHEDULED_AT", value='{{ next_execution_date }}'),
```
Comment (smarchint) [inline ext/scheduler/airflow/dag/expected_dag.2.1.py:105]: this needs to be replicated to 2.4 and 2.6 templates !

Comment (Mryashbhardwaj) [reply ext/scheduler/airflow/dag/expected_dag.2.1.py:105]: they are using `data_interval_end`

Comment (Mryashbhardwaj) [reply ext/scheduler/airflow/dag/expected_dag.2.1.py:105]: ohh okay, then ill remove this

Comment (Mryashbhardwaj) [reply ext/scheduler/airflow/dag/expected_dag.2.1.py:105]: have removed this from 2.4 and 2.6 as well, can you check

**Review by smarchint — APPROVED** [review]
LGTM!

**Pull request merged**